graph/connected_components.cpp

```diff
@@ -1,29 +1,87 @@
+/**
+ *
+ * \file
+ * \brief [Connected Components
+ * (Connected
+ * Components)](https://en.wikipedia.org/wiki/Component_(graph_theory))
+ *
+ * \author [Ayaan Khan](http://github.com/ayaankhan98)
+ *
+ * \details
+ * A graph is a collection of nodes also called vertices and these vertices
+ * are connected by edges. A connected component in a graph refers to a set of
+ * vertices which are reachable form one another.
+ *
+ * Example - Here is graph with 3 connected components
+ *
```

Comment:
this would not get formatted correctly. you probably would have to put it under the HTML `<pre>` tags.

Reply:
okay